www/source/partials/docs/_using-builder-access-tokens.html.md.erb

```diff
@@ -0,0 +1,58 @@
+## <a name="builder-token" id="builder-token" data-magellan-target="builder-token">Create an Access Token</a>
+
+You can build and run Habitat packages without authenticating to Builder; however some operations, like uploading your packages to Builder, or checking status of your build jobs from the Habitat client, will require an access token.
+
+From the Builder site, select your user icon in the upper right-hand side, and select **Profile**.
+
+<img src="/images/screenshots/profile.png">
+
+At the bottom of the profile page, select **Generate Token**.
+
+<img src="/images/screenshots/generate-token.png">
+
+Copy your token from this page.
+
+### Windows
```

Comment:
I question whether any of the below is necessary. How to set environment variables is really outside the scope of habitat documentation. I think most any developers using habitat will be familiar with these sort of details on their platforms. For the sake of completeness, we could link to some other documentation about setting environment variables, but I don't think we should write these docs ourselves.

Reply:
We're trying to make the Windows path as smooth as possible.